.github/workflows/build-images-ci.yaml

```
@@ -22,7 +22,9 @@ on:
     types:
      - completed
 
-permissions: read-all
+permissions:
+  contents: read
+  id-token: write
```

As discussed on #21374:
```suggestion
permissions:
  # To be able to access the repository with actions/checkout
  contents: read
  # Required to generate OIDC tokens for `sigstore/cosign-installer` authentication
  id-token: write
```